wallet/package.json

```diff
@@ -104,6 +104,7 @@
   "devDependencies": {
     "@types/chrome": "^0.0.158",
     "@types/react-copy-to-clipboard": "^5.0.1",
+    "@welldone-software/why-did-you-render": "^6.2.1",
```

Comment:
do we still want this in? / is this triggered somewhere?

Reply:
let's keep it until next week. not triggered in this PR I just forgot to leave out this file.